osx/src/main.cpp

```diff
@@ -278,6 +278,7 @@ void init_main_window(bool recreate) {
 
     // Setup tangram
     Tangram::initialize(sceneFile.c_str());
```

Comment:
I am leaving this as it is right now. Even though `_scenePath` has no purpose in `Tangram::initialize` now and is redundant. But this will save some extra commits, since this code is being re-factored already to handle multiple tangram instances.